models/migrations/v81.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+func changeU2FCounterType(x *xorm.Engine) error {
+	type U2FRegistration struct {
+		Counter uint32 `xorm:"BIGINT"`
+	}
+
+	if err := x.Sync2(new(U2FRegistration)); err != nil {
```

Comment:
This won't fix existing database. You need to do manual column type change:
For postgres:
```
ALTER TABLE `u2f_registration` ALTER COLUMN `counter` TYPE bigint;
```
For MySQL:
```
ALTER TABLE `u2f_registration` MODIFY COLUMN `counter` BIGINT;
```
Microsoft SQL Server:
```
ALTER TABLE `u2f_registration` ALTER COLUMN `counter` bigint;
```
For SQLite you need to do in transaction:

* Rename old table:
```
ALTER TABLE `u2f_registration` RENAME TO `u2f_registration_old`;
```
* Create new table: `sess.Sync2(new(U2FRegistration))`
* Copy data from old to new one:
```
INSERT INTO `u2f_registration` SELECT * FROM `u2f_registration_old`;
```
* Drop old table
```
DROP TABLE IF EXISTS `u2f_registration_old`;
```